lib/assessment.rb

```diff
@@ -20,21 +20,21 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    odd?
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    even?
```

Comment:
As above, `even?` is one of the right tools for the job, but you're missing a key piece. Which enumerable method might we use to check if _every_ element passes a given test?